test/unit/api/index.js

```diff
@@ -548,4 +546,50 @@ describe('api/index.js', () => {
       });
     });
   });
+
+  describe('userActivityReport', () => {
```

Comment:
Could you add the parens to the method? Just so the style matches the rest of the file.
i.e.
```
describe('userActivityReport()', () => {
```